axelrod/random_.py

```diff
@@ -31,6 +31,12 @@ def random_choice(p: float = 0.5) -> Action:
     return Action.D
 
 
+def random_flip(action, threshold):
+    if random.random() < threshold:
```

Comment:
Could we do something similar to `random_choice` so that if `threshold in [0, 1]` we don't sample a random number.

Reply:
Sure

Reply:
Actually we can just reuse `random_choice`